mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -320,6 +320,7 @@ class LocalLDAModel private[spark] (
 
         docBound
       }.sum()
+    ElogbetaBc.destroy(blocking = false)
```

Comment:
`getTopicDistributionMethod` method is only used in `ml.clustering.LDA#transform`, so I think this maybe useful if the model size is large.